packages/client/.eslintignore

```diff
@@ -0,0 +1,3 @@
+__generated__/*
+types/graphql.ts
+serviceWorker/*
```

Comment:
`.eslintignore` in root of parabol project dir was not working, so i broke it out into separate client and server files and put them in the root of their respective packages.